README.md

```diff
@@ -4,34 +4,58 @@ Experimental support for VS Code Server in NixOS. The NodeJS by default supplied
 
 ## Installation
 
+### Flake
 ```nix
 {
-  imports = [
-    (fetchTarball "https://github.com/msteen/nixos-vscode-server/tarball/master")
-  ];
-
-  services.vscode-server.enable = true;
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixos-vscode-server.url ="github:iosmanthus/nixos-vscode-server/add-flake";
+  };
+
+  outputs = inputs@{self, nixpkgs, ...}: {
+    nixosConfigurations.some-host = nixpkgs.lib.nixosSystem rec {
+        system = "x86_64-linux";
+        # For more information of this field, check:
+        # https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/eval-config.nix
+        modules = [
+          ./configuration.nix
+          {
+            imports = [ inputs.auto-fix-vscode-server.nixosModules.system ];
+            services.auto-fix-vscode-server.enable = true;
+          }
+        ];
+      };
+  };
```

Comment:
Is the intention for the user to put this entire flake definition into their `/etc/nixos/configuration.nix`? This seems much more verbose than `services.vscode-server.enable = true;` with an import.

Also will the old import syntax still work for people who aren't on flakes yet? Even if a user has flakes enabled but already has nixos-vscode-server, it seems like we shouldn't brick their installs.

Reply:
1. It intends to enable the user to import this module in their `flake.nix` instead of `configuration.nix`.
2. Yes, you can also use the old syntax to install this module because the `default.nix` is still there.